.github/workflows/e2e.yml

```diff
@@ -0,0 +1,36 @@
+# name: Cypress Run
+# on: [push, pull_request]
```

Comment:
@yichiehliu Will this work? If not we should not include that in PR as it creates failed runs in PRs

Reply:
ok, will remove this